source/VSTSExtensions/OctopusBuildAndReleaseTasks/Tasks/Push/Octopus-Push.ps1

```diff
@@ -7,12 +7,19 @@ try {
 
     . .\Octopus-VSTS.ps1
 
+    $IgnorePullRequestMergeBuilds = Get-VstsInput -Name IgnorePullRequestMergeBuilds -AsBool
     $OctoConnectedServiceName = Get-VstsInput -Name OctoConnectedServiceName
     $ConnectedServiceName = Get-VstsInput -Name ConnectedServiceName
     $Packages = Get-VstsInput -Name Package -Require
     $AdditionalArguments = Get-VstsInput -Name AdditionalArguments
     $Replace = Get-VstsInput -Name Replace -AsBool
 
+    if( $IgnorePullRequestMergeBuilds -and ${env:BUILD_SOURCEBRANCH} -match "^refs/pull/.*/merge$")
```

Comment:
nit:  I think that you may have intended the space to be between the `if` and parenthesis for consistency with surrounding code.